package errors

func CheckInitError(err error) {
	// 临时
	panic(err)
}
